src/middlewared/middlewared/etc_files/systemd/system/getty@ttS1.service.d/override.conf

```diff
@@ -0,0 +1,9 @@
+<%
+	advanced = middleware.call_sync('system.advanced.config')
+	args = f'-l /etc/netcli -n -J'
+	if advanced['serialconsole']:
+	    args += f' {advanced["serialspeed"]}'
+%>\
+[Service]
+ExecStart=
+ExecStart=-/sbin/agetty ${args} ttS1
```

Comment:
I am not familiar with ttS1? What is it? Is it possible a typo for ttyS1? And shouldnt it be ttyS0 if we want it to show for the serial console?

Reply:
I haven't tested serial consoles yet, and yes it's a typo. It should be `ttyS0` or `ttyS1` ( will confirm ), i'll have it corrected.